### ISSUE TITLE
Clarification on PDBbind 2013 core set subset selection

### ISSUE THREAD

**Issue by DynaX-C** [guaguabujianle/GIGN#9]
Hi!

I noticed that the PDBbind 2013 core set contains 195 complexes in the original release, but the paper reports only 107 complexes. Could you clarify how this subset was selected and what filtering steps were applied?

Thanks!

**Comment by guaguabujianle** [comment]
The 2013 test set partially overlaps with other benchmark test datasets; the overlapping structures were therefore removed.

**Comment by DynaX-C** [comment]
There still seems to be some overlap with other benchmark test datasets. It appears that the filtering mainly removed complexes overlapping with the training set. Am I correct in understanding that you may have followed a data processing procedure similar to that used in [OnionNet](https://pubs.acs.org/doi/10.1021/acsomega.9b01997)?

“The core set (v2013), or the CASF-2013 benchmark, a subset of the PDBbind database v2013, which was selected by Li et al., (7) selects PDB complexes after clustering and is primarily used for validating docking scoring function and the CADD benchmark. To compare the performance of our model with other scoring functions conveniently, we prepared a second test set containing 108 complexes from the v2013 core set by removing the overlapping complexes adopted in the validating and training sets. The second test set (called the v2013 core set hereafter) is found to be the subset of the v2016 core set (first test set).”

**Comment by guaguabujianle** [comment]
This work was conducted several yeas ago, and I can't remember the details.

**Comment by DynaX-C** [comment]
Totally understand, thanks for getting back to me anyway.
